include/CLI/Validators.hpp

```diff
@@ -32,37 +32,75 @@ class Option;
 
 ///
 class Validator {
-    friend Option;
-
   protected:
-    /// This is the type name, if empty the type name will not be changed
-    std::string tname;
+    /// This is the validator description
+    std::string description_;
 
-    /// This is the type function, if empty the tname will be used
-    std::function<std::string()> tname_function;
+    /// This is the description function, if empty the description_ will be used
+    std::function<std::string()> desc_function;
```

Comment:
Please use the convention that member variables end in underscore for `desc_function` and `func`.